source/navigation.rst

```diff
@@ -7,8 +7,18 @@ ROS Navigation Stack. A number of tutorials related to navigation
 can be found in the documentation on the
 `ROS Wiki <http://wiki.ros.org/navigation>`_.
 
-Running Navigation in Gazebo
-----------------------------
+Running Navigation: Robot vs Simulation
+---------------------------------------
+
+To start navigation services on the robot you will need to launch the navigation launch file. There is one version of this file for a real robot and another for running in simulation.
+
+To run navigation on a fetch robot, launch the navigation launch file
+from the ``fetch_navigation`` package:
+
+::
+
+    >$ roslaunch fetch_navigation fetch_nav.launch
```

Comment:
in order for this to work -- you need to actually have built a map (hence the way this document reads right now -- where it tells you in the next section that "Gazebo depends on a pre-built map... in order to navigate in the real world, you first have to build a map"